src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -149,22 +156,30 @@ public void setVelocity(double velocity) {
             double volts = controller.calculate(getVelocity(), velocity);
             mainMotor.setVoltage(volts + feedforward.calculate(0, 0));
         } else {
-            mainMotor.set(ControlMode.Velocity, unitModel.toTicks100ms(velocity));
+            mainMotor.set(ControlMode.Velocity, unitModelPosition.toTicks100ms(velocity));
         }
     }
 
+
+    public void setAngleZero() {
```

Comment:
documentation

Reply:
Especially document what zero means (e.g. parallel or perpendicular to the field).